reinforcement_learning/rlclientlib/schema/RankingEvent.fbs

```diff
@@ -0,0 +1,22 @@
+﻿// EventHubInteraction Schema used by FlatBuffer
+
+namespace VW.Events;
+
+// struct has better performace than table. But use table here in case format changes in future.
+table VWStateType {
```

Comment:
>VWStateType [](start = 6, length = 11)

Can we use c++ style naming convention please?  This library is external facing and would be good to not have to switch to different styles for generated code.